docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -162,3 +162,19 @@ test('Close a specific window', async t => {
     await t.closeWindow(window1);
 });
 ```
+
+## Limitations
+
+This section describes the limitations of multiple browser windows testing in TestCafe.
+
+### Limited Video Recording
+
+When you launch TestCafe with [video recording](./screenshots-and-videos.md) enabled, the resulting video scales to the initial (parent) browser window.
```

Comment:
Sorry if I sound nerdy, but there's no resulting video when you launch TestCafe... yet.

It's rather "recording starts inside (is framed to) the initial browser window"